### DISCUSSION TITLE
Mind Escape testing elements

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4148] [Feature Suggestions]
First test element is standing Energy pellet catcher(?)
![image](https://user-images.githubusercontent.com/82114463/169250815-0d7f8f6e-2aaf-486a-a167-a98f1d7e3ee8.png)
You can change timer for him and wire to button that will open/close the catcher.

Also next test element is magnifying button(cubue balls etc)
You can enable and disable the magnifying function in the button when you wiring it to button
![image](https://user-images.githubusercontent.com/82114463/169251820-85b8dfa9-9def-441e-a81f-32caabe5e47b.png)

**Comment by slambo311** [comment]
Areng's UCP already has something similar to the magnifying button, but the standing energy pellet catcher would be nice.

**Comment by unfunny1** [comment]
we need it
